pkg/kubectlcmd/client.go

```diff
@@ -48,22 +48,38 @@ func (console) Run(c *exec.Cmd) error {
 }
 
 // kubectlParams is a set of params passed to kubectl.
+// dryRun - display the command but don't run it
+// verbose - dump the full manifest
+// kubeconfig, context - used to identify the cluster
+// namespace - k8s namespace for kubectl command
```

Comment:
you don't need to repeat this since params are already annotated below.

Reply:
updated. my bad.